File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/proposal/GraduationDesignProposalAddVo.java
package top.zbeboy.isy.web.vo.graduate.design.proposal;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by lenovo on 2017-06-25.
 */
@Data
public class GraduationDesignProposalAddVo {
    @Size(max = 10)
    private String version;
    @NotNull
    private Integer graduationDesignDatumTypeId;
    @NotNull
    @Size(max = 64)
    private String graduationDesignReleaseId;
    private String graduationDesignDatumId;
}

File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/proposal/GraduationDesignProposalAddVo.kt
package top.zbeboy.isy.web.vo.graduate.design.proposal

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2018-01-29 .
 **/
open class GraduationDesignProposalAddVo {
    @Size(max = 10)
    var version: String? = null
    @NotNull
    var graduationDesignDatumTypeId: Int? = null
    @NotNull
    @Size(max = 64)
    var graduationDesignReleaseId: String? = null
    var graduationDesignDatumId: String? = null
}